docs/Compiler Breaking Changes - DotNet 8.md

```diff
@@ -3,7 +3,7 @@ title: Razor compiler breaking changes since .NET 8
 description: Learn about any breaking changes since the initial release of .NET 8.
 author: rick-anderson
 ms.author: riande
-ms.date: 2024-11-12
+ms.date: 11-12-2024
```

Comment:
Use ISO date formats! Is this november 12, or december 11th!

Reply:
I've made that request of the platform team.